muhasin/index.js

```diff
@@ -0,0 +1,18 @@
+//1.Declare a variable and assign a string as a value. Make sure the string is at least 6 characters long. Print the variable to the console.
+let str = "webdevelopment";
+console.log("str ==>", str);
+
+// 2. Access the 2nd and 6th elements of the string. Print the elements to the console. Make sure you have selected the right index numbers
+console.log("str ==>", str[2], str[6]);
```

Comment:
The index of second element is 1 and index of 6th element is 5.
console.log("str ==>", str[1], str[5]);